src/Tasks/MigrateContentToElement.php

```diff
@@ -2,13 +2,50 @@
 
 namespace DNADesign\Elemental\Tasks;
 
+use DNADesign\Elemental\Extensions\ElementalAreasExtension;
+use DNADesign\Elemental\Extensions\ElementalPageExtension;
+use DNADesign\Elemental\Models\BaseElement;
 use DNADesign\Elemental\Models\ElementalArea;
 use DNADesign\Elemental\Models\ElementContent;
-
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\Core\Injector\Injector;
 use SilverStripe\Dev\BuildTask;
+use SilverStripe\Versioned\Versioned;
 
 class MigrateContentToElement extends BuildTask
 {
+    /**
+     * Configures if the existing content should be left blank.
```

Comment:
```suggestion
     * Configures if the existing content should be cleared once the migration task has completed.
```

I found the "should be left blank" part a little ambiguous